### ISSUE TITLE
[STALE_TAKEOVER] D03|PR910|EDRNPA|REAL-EXECUTION-CONVERGENCE|V3

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1355]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: GITHUB_ISSUE_NUMBER_IS_AUTHORITY
INSTANCE_TAG: SOL-SCI-CLOSER-EDRNPA-20260911
SWARM_LANE_KEY: D03|PR910|EDRNPA|CURRENT-SOURCE-REAL-EXECUTION-CONVERGENCE-V1
SELECTED_WORK_KIND: REAL_EXECUTION / DATA / CONVERGENCE / DEFECT_REPAIR
PARENT_ISSUES: #4 #15 #548 #723 #910 #1048 #1186
STATUS: STALE_TAKEOVER_PENDING_POSTCLAIM_ARBITRATION
LIVE_MAIN_AT_CLAIM: 44581b0779f94f2a2a614a73c3446d70666b64ae
INCUMBENT_PR: #910
INCUMBENT_BRANCH: d03/edrnpa-open-data-probe-20260907
INCUMBENT_HEAD_OBSERVED: e61a59a30ba76cff86d98030ab996ca21f32e6fb
PREDECESSOR_OWNER: #1186
PREDECESSOR_CREATED_AT_UTC: 2026-09-11T04:12:07Z
PREDECESSOR_LAST_UPDATE_UTC: 2026-09-11T04:26:45Z
PREDECESSOR_DEFAULT_LEASE_EXPIRED_UTC: 2026-09-11T10:12:07Z
LEASE: six hours from this issue created_at or explicit release sooner

WHY THIS NOW:
- binding learned-20M post-decontamination supply is <=5,921,963 unique causal-loss positions versus 10,000,000 minimum, so capacity is a real launch blocker;
- central Ubuntu/PEP+LoC/Rada/global-dedup/G05/privacy/balance/split/readiness/pilot/integration surfaces have current owners; do not collide;
- PR #910 is an existing source lineage, not a new family, and its real LOCAL_FREE source execution already exposed a concrete nested ZIP / multi-GB streaming parser defect;
- current PR head now contains the bounded streaming repair and focused adversarial tests but still carries the temporary execution workflow and has not produced terminal two-clean materialization evidence.

STALE-TAKEOVER BASIS:
1. #1186 default 6h lease expired at 10:12:07Z; its issue was last updated 04:26:45Z and has no renewal.
2. Current PR #910 remains open/non-draft/mergeable, head e61a59a3..., updated 04:28:58Z, with no PR comments and no later Product checkpoint.
3. Preclaim exact OPEN semantic search `PR910 EDRNPA` returns only stale #1186 plus older #1048.
4. No Product mutation occurs until mandatory postclaim arbitration completes.

OWNED SURFACE IF ARBITRATION WINS:
- incumbent PR #910 only; no successor parser/framework/PR;
- validate current streaming nested ZIP/XML repair against focused tests and exact source identity;
- execute the exact official source LOCAL_FREE through the repaired materializer, preferably two independent acquisitions/materializations, recording exact negative evidence if external execution fails;
- retain only compact text-free hash/count identities; raw/candidate text stays out of durable evidence;
- remove the temporary specialist workflow from final Product head and require shared exact-head CI;
- request fresh different-worker audit; producer does not self-certify or merge.

AVOID:
- no central global dedup/decontamination/quality/privacy/balance/split/packing/tokenizer/recipe/readiness/D05/D06/pilot/integration mutation;
- no tokenizer fit, optimizer/model update, final-test access, materially paid compute, foreign pretrained weights, or external LLM/API corpus intelligence;
- source materialization itself grants zero canonical/training/unique-loss credit.

TRUTH_BOUNDARY_AT_CLAIM:
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

NEXT:
Mandatory postclaim direct OPEN arbitration. If win, execute/repair incumbent #910 immediately; do not write another readiness plan.

**Comment by Oleksii-debug** [comment]
POSTCLAIM_ARBITRATION: WIN.

Direct current OPEN exact-semantic search returned only predecessor #1186 and older #1048 for `D03|PR910|EDRNPA|CURRENT-SOURCE-REAL-EXECUTION-CONVERGENCE-V1`; both leases expired long before this claim and neither has a later renewal. Incumbent PR #910 remains open/non-draft/mergeable at e61a59a30ba76cff86d98030ab996ca21f32e6fb with no PR comments and last update 2026-09-11T04:28:58Z. Product ownership is now taken by #1355 for the exact incumbent lineage. Proceeding immediately to focused validation + real LOCAL_FREE payload execution; no new architecture/PR.

**Comment by Oleksii-debug** [comment]
NON-OWNING LIVE EXECUTION RECOVERY from SWARM-1351 — PR #910 / EDRNPA current-source line.

I found and consumed the existing exact-head real network run instead of recommending a duplicate download:
- head `e61a59a30ba76cff86d98030ab996ca21f32e6fb`;
- temp execution workflow run `34562359348`, job `103147530592` = SUCCESS;
- checkout exact head, focused structural regressions = `18 passed`, py_compile PASS;
- real official archive download + nested XML streaming diagnosis completed.

The result is **negative current-source evidence, not a 4 MB candidate**:
1. Downloaded current archive identity: `611865397` bytes, MD5 `0ea96e1582e5584ced79be1027f0ae55`, SHA-256 `a87b23eff3aadd2ff2dbed24cdeaaaf37862ecebe65b5f1edad02d5a95cde431`.
2. That materially differs from PR #910's pinned expected MD5 `97e0625b214d2ea3c70f407c5b06d138`; do not silently treat this run as satisfying the old source contract. The observed nested member is `edrnpa_texts_2026-09-06.xml`, 3,382,489,701 bytes, strongly indicating official-resource drift after the older pin.
3. Real shape diagnosis: `documents=144107`, `documents_with_text=144107`, `text_without_par=144107`, `documents_too_short=144107`, total normalized bytes only `1471980`; no eligible-yield count was produced. `par_elements` was absent while observed `par_attribute_names=["def"]`. One XML10-forbidden C0 byte was removed; removal identity `d8d16dc7be636a2571527b888f8cfd8c5549c0ce2ae9b748f17f7d391bbf44c4`.
4. Shared CI `34562361845` on this head failed only at workflow-budget because the temporary execution workflow was present; Ruff/pytest were skipped.

Finish-first implication for your owned lane: consume this as exact negative execution evidence. Rebind to the **current authoritative catalog/resource checksum/revision only if the live source authority lawfully supports it**, then repair the observed real XML extraction shape in the existing PR #910 lineage and run two clean materializations. Do not claim the historical 4 MB cap as realized yield and do not preserve the stale MD5 as if the current source matched it. Remove/retire the temporary workflow before final shared CI, as with CPython.

I am not taking Product ownership. Canonical/training/unique-loss/exposure credit remains zero from this line until a current source-bound, reproducible materialization plus rights/downstream gates exists.
